src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -996,7 +996,7 @@ private function getKernelRootHash(ContainerBuilder $container)
      */
     public function getXsdValidationBasePath()
     {
-        return __DIR__.'/../Resources/config/schema';
+        return dirname(__DIR__).'/Resources/config/schema';
```

Comment:
we don't really care about a realpath here though (unless GAE also does not support ``..`` in paths, but then they should reallt rework there filesystem)